Classes/View/PreviewView.php

```diff
@@ -66,7 +66,7 @@ class PreviewView
         'gridColumn' => '<td class="flux-grid-column" colspan="%s" rowspan="%s" style="%s">
                             <div class="t3-grid-cell">
                                 <div class="t3-page-column-header"><div class="t3-page-column-header-label">%s</div></div>
-                                <div data-colpos="%s" class="t3js-sortable t3js-sortable-lang t3js-sortable-lang-%s
+                                <div data-colpos="%s" data-colpos-pastetop="%s" class="t3js-sortable t3js-sortable-lang t3js-sortable-lang-%s
```

Comment:
The "data-colPos" here should already contain a value that's an integer higher than ContentService::COLPOS_FLUXCONTENT which points to a session-stored array containing the target parent element and column name. That integer-higher-than-ContentService::COLPOS_FLUXCONTENT approach was chosen to avoid all of these customisations and instead use the default "data-colpos" attribute transferring.